packages/cxl-ui/scss/global/cxl-card.scss

```diff
@@ -0,0 +1,56 @@
+cxl-card {
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  margin: var(--lumo-space-m);
+  overflow: hidden;
+  text-overflow: ellipsis;
+  background: var(--lumo-base-color);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius);
```

Comment:
Why do we style base card globally, instead of shadow?

All card content, yes, must be global. But card chrome itself?